HPHBhelper/hogwarts/templates/HPHBhelper/index.html

```diff
@@ -7,7 +7,7 @@
   <h2>Active games</h2>
 <ul>
       {% for game in active_games %}
-          <li><a href="/games/{{ game.shortcode }}/">{{ game.shortcode }}</a></li>
+          <li><a href="/hogwarts/games/{{ game.shortcode }}/">{{ game.shortcode }}</a></li>
```

Comment:
use Django to construct the URL, e.g. `<a href="{% url 'bites:index' %}">Keep calm and code in Python!</a>`